pkg/integration/utils_test.go

```diff
@@ -234,7 +234,7 @@ func TestConsumeWithSpeed(t *testing.T) {
 	reader := strings.NewReader("1234567890")
 	chunksize := 2
 
-	bytes1, err := ConsumeWithSpeed(reader, chunksize, 1*time.Second, nil)
+	bytes1, err := ConsumeWithSpeed(reader, chunksize, 10*time.Millisecond, nil)
```

Comment:
@cpuguy83 I do agree on "I'd prefer a test that takes a little longer to fail than a flakey test.", 1 second might be still ok but it's already long for a unit test (we could flag them though).